index.js

```diff
@@ -152,7 +152,11 @@ exports.handler = function(event, context) {
             // Migration code for old collectors.
             // This needs to be done because the collector lambda does not have premissions to set its own env vars.
             if(!process.env.aws_lambda_update_config_name){
-                process.env.aws_lambda_update_config_name = 'configs/lambda/config.json';
+                process.env.aws_lambda_update_config_name = 'configs/lambda/al-cwe-collector.json';
+            }
+            //add in the env var for the framework
+            if(!process.env.stack_name && event.StackName){
+                process.env.stack_name = event.StackName;
```

Comment:
Let's separate this code into a function and call it even before `async.waterfall` is called.